### PR TITLE
Fix C API for Visual Studio

### PR DESCRIPTION
**Pull request by SpaceIm**
closes https://github.com/libspatialindex/libspatialindex/issues/179

### CODE REVIEW

**Comment by hobu** [comment]
Can you provide more detail why this change was needed?

File: include/spatialindex/capi/sidx_config.h [hunk 1]
```diff
@@ -42,10 +42,8 @@
   typedef unsigned __int16 uint16_t;
   typedef unsigned __int32 uint32_t;
   typedef unsigned __int64 uint64_t;
-#endif
-
-#if _MSC_VER >= 1900
-   #include <cinttypes>
+#else
+  #include <stdint.h>
```
Comment (mortezaipo) [inline include/spatialindex/capi/sidx_config.h]: use `cstdint` instead.

Comment (SpaceIm) [reply include/spatialindex/capi/sidx_config.h]: cstdint is a C++ header. Why do you want to include a C++ header in a C API?

**Comment by SpaceIm** [comment]
Because cinttypes is a C++ header, therefore a pure C program will fail to compile with msvc.

**Pull request merged**